Tella/Data/Networking/Upload/BaseUploadOperation.swift

```diff
@@ -474,32 +478,18 @@ extension BaseUploadOperation {
     
     func getAPIResponse<Value1, Value2> (response:HTTPURLResponse?, data: Data?, error: Error?) -> UploadDecode<Value1, Value2> where Value1: DataModel, Value2: DomainModel {
         
+        let allHeaderFields = (response)?.allHeaderFields
+        
         guard let code = (response)?.statusCode else {
-            return UploadDecode(dto: nil, domain: nil, error: APIError.unexpectedResponse)
+            return UploadDecode(dto: nil, domain: nil, error: APIError.unexpectedResponse, headers: allHeaderFields)
         }
         guard HTTPCodes.success.contains(code) else {
             debugLog("Error code: \(code)")
-            return UploadDecode(dto: nil, domain: nil, error: APIError.httpCode(code))
+            return UploadDecode(dto: nil, domain: nil, error: APIError.httpCode(code), headers: allHeaderFields)
```

Comment:
@rob-horizontal-iOS Could you centralise the error codes in enum to avoid the hardcoded values, so you can update the `HTTPCode` type

Reply:
@dhekra-rouatbi  Centralized the error codes in enum `HTTPErrorCodes` to avoid the hardcoded values